Tests/Svg.UnitTests/Svg.UnitTests.csproj

```diff
@@ -36,11 +36,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />

```

Comment:
16.8.3 has a security vulnerability so I upgraded to 16.11.0 to silence the warning.